realm-transformer/src/main/java/io/realm/transformer/RealmAnalytics.java

```diff
@@ -68,6 +68,7 @@
             + "      \"Anonymized MAC Address\": \"%USER_ID%\",\n"
             + "      \"Anonymized Bundle ID\": \"%APP_ID%\",\n"
             + "      \"Binding\": \"java\",\n"
+            + "      \"Uses Kotlin\": \"%USES_KOTLIN%\",\n"
```

Comment:
To detect the usage of Java requires traversing all the project folders, which can be quite slow for big projects. So that leaves us at being able to easily detect pure Java or possibly mixed project. but it would be expensive to compute whether a project is surely mixed or pure Kotlin.